CONTRIBUTORS

```diff
@@ -259,6 +262,7 @@ Lunrong Chen <lunrong@google.com>
 Madhav Sainanee <madhavsainanee123@gmail.com>
 Madiyar Aitbayev <Madiyar92@gmail.com>
 Mahendra Suthar <mahendra.suthar.1290@gmail.com>
+Mai Elshiashi <mai.elshiashi@gmail.com>
```

Comment:
Ditto, OK to delete the line below this one.

Reply:
Done